app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,13 @@
     <string name="reg_vm_valid_email">The Email is being used in another account</string>
 
     <string name="hello_blank_fragment">Hello blank fragment</string>
+    <!-- Google Web client -->
+    <string name="default_web_cliente_id">11042396908-c4tf4cmfen29e6oln7uum134r3vsvo08.apps.googleusercontent.com</string>
```

Comment:
Also, mark it with _translatable="false"_ so we don't add it on all the string files